release-notes/3.1/3.1.1/3.1.1-install-instructions.md

```diff
@@ -20,7 +20,7 @@ Snap is a system which installs applications in an isolated environment and prov
 
 After configuring Snap on your system, run the following command to install the latest .NET Core SDK.
 
-`sudo snap install dotnet-sdk --channel 3.1/stable –classic`
+`sudo snap install dotnet-sdk --channel 3.1/stable -–classic`
```

Comment:
The second `-` in `--classic` is actually a `–` (em-dash?) here. That might confuse anyone who copy/pastes this code. Maybe change both dashes to `-`?

Reply:
I have updated the command to `--classic`